composeApp/src/desktopMain/kotlin/edu/dyds/movies/data/MoviesRepositoryImpl.kt

```diff
@@ -0,0 +1,29 @@
+package edu.dyds.movies.data
+
+import edu.dyds.movies.data.external.ExternalMoviesSource
+import edu.dyds.movies.data.local.LocalMoviesSource
+import edu.dyds.movies.domain.entity.Movie
+import edu.dyds.movies.domain.repository.MoviesRepository
+
+class MoviesRepositoryImpl(private val localMovies: LocalMoviesSource, private val externalMovies: ExternalMoviesSource): MoviesRepository {
+    override suspend fun getPopularMovies(): List<Movie> {
+        val cachedMovies = localMovies.getCachedMovies()
```

Comment:
```suggestion
        val localMovies = localMovies.getCachedMovies()
```
el término cache es detalle de implementación